include/nng/transport/file_descriptor/file_descriptor.h

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2021 Staysail Systems, Inc. <info@staysail.tech>
+// Copyright 2017 Capitar IT Group BV <info@capitar.com>
+//
+// This software is supplied under the terms of the MIT License, a
+// copy of which should be located in the distribution where this
+// file was obtained (LICENSE.txt).  A copy of the license may also be
+// found online at https://opensource.org/licenses/MIT.
+//
+
+#ifndef NNG_TRANSPORT_TCP_TCP_H
```

Comment:
```suggestion
#ifndef NNG_TRANSPORT_FILE_DESCRIPTOR_FILE_DESCRIPTOR_H
```

This seems really long -- I almost think we should use the abbreviation "fd" in most places (not the URL though, as that needs to be sure not to collide with other URL schemes that we don't control.)